### PR TITLE
Teams to support ACAPY transition

### PR DESCRIPTION
**Pull request by ryjones**

### CODE REVIEW

**Comment by ryjones** [comment]
@WadeBarnes

**Comment by ryjones** [comment]
@swcurran I want to be sure these are the new teams and people the project wants

File: config.yaml [hunk 1]
```diff
@@ -1,136 +1,184 @@
...
+      - aholovko
+      - auer-martin
+      - baha-ai
+      - dbluhm
+      - esune
+      - ianco
+      - jamshale
+      - rolsonquadras
+      - troyronda
+      - wadeking98
```
Comment (WadeBarnes) [inline config.yaml:49]: Wade King (`wadeking98`) is no longer with the BC Wallet team.

File: config.yaml [hunk 2]
```diff
@@ -1,136 +1,184 @@
...
+    maintainers:
+      - jleach
+    members:
+      - amanji
+      - bryce-mcmath
+      - cvarjao
+      - jcdrouin21
+      - MosCD3
+      - thiagoromanos
+      - wadeking98
```
Comment (WadeBarnes) [inline config.yaml:63]: Wade King (`wadeking98`) is no longer with the BC Wallet team.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]

**Comment by swcurran** [comment]
Adjustments that I think are needed:

- Remove from all lists:
  - aholovko
  - auer-martin
  - baha-ai
  - dhh1128
  - nage
  - rolsonquadras
  - troyronda
  - wadeking98
- Add 
  - dbluhm to Admins list

**Comment by ryjones** [comment]
@swcurran dbluhm is in the admins list?

File: config.yaml (hunk 2 above)
Comment (ryjones) [reply config.yaml]: done

**Review by WadeBarnes — APPROVED** [review]

**Review by swcurran — APPROVED** [review]
LGTM

**Pull request merged**